### PR TITLE
Testimonials

### PR DESCRIPTION
**Pull request by sayyedarib**
@Illaf @Riyazul555 please improve the UI of testimonials  card

also there's some issue with image size

### CODE REVIEW

**Comment by Riyazul555** [comment]
Ok @anmode I will improve the testimonials and image size

**Comment by Illaf** [comment]
@anmode okay I will work on this along with riyaz

File: components/Testimonial.js
Comment (anmode) [inline components/Testimonial.js:1]: Testimonial component tha... that's cool!

File: pages/[mentorUsername].page.js [hunk 1]
```diff
@@ -111,35 +112,51 @@ function Index({ mentorDetail }) {
...
+            testimonialUserHeadline={mentorDetail.testimonials.headline}
+            testimonialUserImage={mentorDetail.testimonials.image}
+            testimonialRate={mentorDetail.testimonials.rate}
+            testimonialDescription={mentorDetail.testimonials.description} /> */
+            }
+          </div>
+          <div className="tw-w-22 tw-h-auto tw-flex tw-flex-wrap ">
+            {/* {for testing purpose}  */}
+          <Testimonial testimonialUserName="test_user" testimonialUserHeadline="test headline" testimonialRate="4" testimonialUserImage="/assets/img/icon/no-profile-picture.webp" testimonialDescription="jdsfkjksadjfkaf askdjflsadkfk kfas kasjdfk sadklfjsd fs dfljsadfkasdl lorem50" />
+
```
Comment (anmode) [inline pages/[mentorUsername].page.js:158]: hard coded data should be changed! Will do after UI improved

**Review by anmode — COMMENTED** [review]
For testimonial I request to see topmate page:

**Comment by anmode** [comment]
Some testimonials are at home page...that's why please another testimonial component!

**Pull request merged**